app/views/exchanges/hbx_profiles/_family_index_datatable.html.erb

```diff
@@ -7,23 +7,25 @@
 <%= raw render_datatable(@datatable) %>
 
 <div class="container">
-  <div class="col-sm-12 col-md-12">
-  <%=pundit_span Family, :updateable? %>
-    <div class="row">
-         <ul class='col-xs-12 col-md-5 col-lg-5'>
-          <li> <%=link_to 'New Consumer Phone Application', begin_consumer_enrollment_exchanges_agents_path(
-          original_application_type: 'phone')  %> <br/></li>
-
-          <li> <%=link_to 'New Consumer Paper Application', begin_consumer_enrollment_exchanges_agents_path(
-          original_application_type: 'paper')  %> <br/></li>
-
-          <% if policy(:family).hbx_super_admin_visible? %>
-            <li> <%=link_to 'New DC Resident Application', begin_resident_enrollment_exchanges_residents_path(
-                 original_application_type: 'paper')  %> </li>
-          <% end %>
-          </ul>
-    </span>
-    </div>
+  <% if (Settings.aca.individual_market.is_active) %>
```

Comment:
I have the same desire in the views - define a method in a separate helper file to do this logic and use that instead.

<% if individual_market_is_enabled? %> etc etc